lychee-lib/src/extract.rs

```diff
@@ -122,15 +115,11 @@ fn walk_html_links(mut urls: &mut Vec<String>, node: &Handle) {
 fn elem_attr_is_link(attr_name: &str, elem_name: &str) -> bool {
     // See a comprehensive list of attributes that might contain URLs/URIs
     // over at: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
-    matches!(
-        (attr_name, elem_name),
-        ("href", _)
-            | ("src", _)
-            | ("srcset", _)
-            | ("cite", _)
-            | ("data", "object")
-            | ("onhashchange", "body")
-    )
+    matches!(attr_name, "href" | "src" | "srcset" | "cite")
```

Comment:
Kind of find the older version more readable here. 
Instead of two matches I prefer the redundant `_`, because it indicates that the second value in the tuple is insignificant for that match (e.g. in the case of `("href", _)`).
It's highly subjective, though.

Reply:
```rust
    matches!(
        (attr_name, elem_name),
        ("href", _)
            | ("src", _)
            | ("srcset", _)
            | ("cite", _)
            | ("data", "object")
            | ("onhashchange", "body")
    )
```

yes, looks a bit clearer.